src/app/person/person.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <memory>
+#include <string>
+
+class Person
+{
+    std::string firstName_;
+    std::string lastName_;
+    std::string pesel_;
+    
+  public:
+    Person(std::string firstName, std::string lastName, std::string pesel);
```

Comment:
Will default ctor be accepted? 
If not then please make it 
`Person() = delete;`
Also if we will have only one ctor, use explicit
`explicitPerson(std::string firstName, std::string lastName, std::string pesel);`

Reply:
For now, we'll maintain just one constructor for the Person class, though this approach might be subject to change in the future.